test/plain-list/AutosuggestApp.test.js

```diff
@@ -456,6 +803,19 @@ describe('Plain list Autosuggest', () => {
       });
     });
 
+    it('should not call onChange when Enter is pressed and suggestion is focused with down since value did not change', () => {
+      onChange.reset();
+      clickDown();
+      clickEnter();
+
+      // onChange should not have been called because the value didn't change
+      expect(onChange).to.have.been.calledExactlyOnce;
+      expect(onChange).to.be.calledWithExactly(eventInstance, {
+        newValue: 'Java',
+        method: 'down'
+      });
+    });
```

Comment:
That should say `should not call onChange for the 'Enter' event` to be more clear, since `onChange` will be called, and called only for, the `Down` event.   
  
I'll update the wording and move.